src/core/agent/services/identifierService.ts

```diff
@@ -144,6 +148,30 @@ class IdentifierService extends AgentService {
     });
   }
 
+  async getSigner(identifier: string): Promise<Signer> {
+    const metadata = await this.identifierStorage.getIdentifierMetadata(
+      identifier
+    );
+    this.validIdentifierMetadata(metadata);
+
+    if (!metadata.signifyName) {
+      throw new Error(
+        `${IdentityWalletConnect.AID_MISSING_SIGNIFY_NAME} ${metadata.id}`
+      );
+    }
```

Comment:
We can actually remove all of the checks for `signifyName` and the `AID_MISSING_SIGNIFY_NAME` error now. Since we removed did:key, all identifiers have a signify name now and it's no longer optional.